packages/zwave-js/src/lib/telemetry/metrics.ts

```diff
@@ -0,0 +1,90 @@
+import { formatId } from "@zwave-js/shared";
+import { isObject } from "alcalzone-shared/typeguards";
+import axios from "axios";
+import * as crypto from "crypto";
+import type { Driver } from "../driver/Driver";
+
+const baseUrl = "https://metrics.zwave-js.io:9998"; // No leading slash!
```

Comment:
why on a non standard port? you may find the client gets blocked or you just draw unnecessary attention from any network DLP/IDP type thing

Reply:
gonna change that to `https://stats.zwave-js.io` without port